<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HealthCareWorkerAuthPage</class>
 <widget class="QWidget" name="HealthCareWorkerAuthPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Electronic Vaccination Data System</string>
  </property>
  <property name="inputMethodHints">
   <set>Qt::ImhDigitsOnly|Qt::ImhSensitiveData</set>
  </property>
  <widget class="QLabel" name="lbl_auth">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>90</y>
     <width>201</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Professsional Registration Number</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="et_hcwPRN">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>120</y>
     <width>201</width>
     <height>22</height>
    </rect>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhHiddenText|Qt::ImhNoAutoUppercase|Qt::ImhNoPredictiveText|Qt::ImhSensitiveData</set>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_profRegSubmit">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>160</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>S u b m i t</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_rtnHme">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="toolTip">
    <string extracomment="Return To Home Page">Home</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../Res/home.png</normaloff>../Res/home.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="lb_hcwNotice">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>220</y>
     <width>271</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Incorrect PRN Entered, please try again.</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
